tldw_Server_API/app/core/Jobs/manager.py

```
@@ -257,6 +263,66 @@ def _pg_cursor(self, conn):
+                f"JOBS_{backend.upper()}_ACQUIRE_TIE_BREAK_{dom.upper()}",
+                f"JOBS_{backend.upper()}_ACQUIRE_TIE_BREAK",
+                f"JOBS_ACQUIRE_TIE_BREAK_{dom.upper()}",
+                "JOBS_ACQUIRE_TIE_BREAK",
+            ]
+            for k in cands:
+                v = os.getenv(k)
+                if v:
+                    v2 = v.strip().lower()
+                    if v2 in {"fifo", "lifo"}:
+                        return v2
+            return None
```

Ruff (0.14.3)

Consider moving this statement to an `else` block

(TRY300)